docs/release_docs/Dev-ci-cd_ReleaseFlow.md

```diff
@@ -0,0 +1,18 @@
+## Dev-CI-CD
```

Comment:
The CI-CD process is mandatory, not an optional choice, and our sentences should reflect the same concept.

Re-frame it as: "Dev Workflow is triggered automatically whenever any PR is merged into the `develop` branch. The workflow builds the applications and deploys the changes to the RF Platform Development environment."